pkgs/by-name/ni/nickel/package.nix

```diff
@@ -25,10 +29,21 @@ rustPlatform.buildRustPackage rec {
     "-p nickel-lang-lsp"
   ];
 
-  nativeBuildInputs = [
-    python3
+  buildInputs = lib.optionals enableNixImport [
+    nix
+    boost
   ];
 
+  nativeBuildInputs =
+    [
+      python3
+    ]
+    ++ lib.optionals enableNixImport [
+      pkg-config
+    ];
+
+  buildFeatures = lib.optionals enableNixImport [ "nix-experimental" ];
+
   outputs = [
     "out"
     "nls"
```

Comment:
The sed in preBuild should be a subsituteInPlace

Reply:
thanks, i adjusted it now